src/js/components/devices/device-details/devicetwin.js

```diff
@@ -130,7 +131,13 @@ export const DeviceTwin = ({ device, getDeviceTwin, integrations, setDeviceTwin
   const [isSync, setIsSync] = useState(true);
   const editorRef = useRef(null);
 
-  const integration = integrations.find(integration => integration.provider === externalProvider.provider);
+  var integrationIds = Object.keys(device.twinsByIntegration ?? {}).filter(n => n != 'twinError');
```

Comment:
will the device twin really be populated by a `twinError`?

Reply:
no, it is populated by the UI